pallets/laos-evolution/src/benchmarking.rs

```diff
@@ -142,19 +138,164 @@ mod benchmarks {
 		}
 	}
 
+	#[benchmark]
+	fn precompile_mint() {
```

Comment:
wouldn't be more convenient use linear fn to provide the params as it is done here https://github.com/freeverseio/laos/blob/feature/precompile_benchmarks/pallets/laos-evolution/src/benchmarking.rs#L314 ?